project-koku/koku
# catch integrityerror during source delete

This PR catches some 500s that are raised when trying to delete a Source that is ingesting data. Essentially the delete process through sources-client will retry until successful. The retry in the sources admin delete will retry 5 times before failing.

Testing admin endpoint:
1. Add our AWS source.
2. While files start ingesting (check worker logs), delete the source thru the admin endpoint (this might take it several seconds)
3. Check koku-server logs and see that the deletion process retried

Testing koku-sources (error while processing the kafka message):
1. Add AWS source thru sources-backend
2. While files start ingesting (check worker logs), delete the source thru the sources-backend
3. Check sources-client logs and see that the deletion process retried
